project/Version.scala

```diff
@@ -1,5 +1,5 @@
 object SpinalVersion {
-  val compiler = "2.11.12"
+  val compiler = "2.12.0"
```

Comment:
Basicaly, from 2.11.x to 2.12.x there is things which break. There is specific SpinalHDL   branch for 2.12.x that i update before each release.
Can you revert that change ?